tests/App/Datagrid/Datagrid.php

```diff
@@ -43,7 +43,7 @@ public function getPager()
         return $this->pager;
     }
 
-    public function getQuery()
+    public function getQuery(): void
```

Comment:
that one is not correct, it is adding void because it is not implemented

Reply:
Oh right guess I went too fast…